packages/cli/src/lib/defaults/deploy-modules/ens-recursive-name-register/getClient.ts

```diff
@@ -0,0 +1,96 @@
+import { ethereumProviderPlugin, Connections } from "ethereum-provider-js";
+import { PolywrapClient } from "@polywrap/client-js";
+import {
+  PackageToWrapperCacheResolver,
+  RecursiveResolver,
+  StaticResolver,
+  WrapperCache,
+} from "@polywrap/uri-resolvers-js";
+import { embeddedWrappers } from "@polywrap/test-env-js";
+import {
+  ClientConfigBuilder,
+  defaultInterfaces,
+  defaultIpfsProviders,
+  defaultPackages,
+  defaultWrappers,
+} from "@polywrap/client-config-builder-js";
+import { ipfsPlugin } from "@polywrap/ipfs-plugin-js";
+import { ipfsResolverPlugin } from "@polywrap/ipfs-resolver-plugin-js";
+import { httpPlugin } from "@polywrap/http-plugin-js";
+import { ensResolverPlugin } from "@polywrap/ens-resolver-plugin-js";
+import { ExtendableUriResolver } from "@polywrap/uri-resolver-extensions-js";
+import { fileSystemPlugin } from "@polywrap/fs-plugin-js";
+import { fileSystemResolverPlugin } from "@polywrap/fs-resolver-plugin-js";
+
+export function getClient(connections: Connections): PolywrapClient {
+  const uriResolver = RecursiveResolver.from(
+    PackageToWrapperCacheResolver.from(
+      [
+        StaticResolver.from([
+          {
+            from: "wrap://ens/uts46.polywrap.eth",
+            to: embeddedWrappers.uts46,
+          },
+          {
+            from: "wrap://ens/sha3.polywrap.eth",
+            to: embeddedWrappers.sha3,
+          },
+          {
+            from: "wrap://ens/ethereum.polywrap.eth",
+            to: defaultWrappers.ethereum,
+          },
+          {
+            uri: defaultPackages.fileSystem,
+            package: fileSystemPlugin({}),
+          },
+          {
+            uri: defaultPackages.fileSystemResolver,
+            package: fileSystemResolverPlugin({}),
+          },
+          {
+            uri: defaultPackages.ipfs,
+            package: ipfsPlugin({}),
+          },
+          {
+            uri: defaultPackages.ipfsResolver,
+            package: ipfsResolverPlugin({}),
+          },
+          {
+            uri: defaultPackages.http,
+            package: httpPlugin({}),
+          },
+          {
+            uri: defaultPackages.ensResolver,
+            package: ensResolverPlugin({}),
+          },
+          {
+            uri: defaultPackages.ethereumProvider,
+            package: ethereumProviderPlugin({ connections }),
+          },
+        ]),
+        new ExtendableUriResolver(),
+      ],
+      new WrapperCache()
+    )
+  );
+  const coreClientConfig = new ClientConfigBuilder(undefined, uriResolver)
```

Comment:
Why aren't we using addDefaults?

Reply:
Fixed! The complicated config without defaults was leftover from when I was trying to fix something, and I forgot to change it back.